dbt/redshift/settings/default.py.schema

```diff
@@ -0,0 +1,78 @@
+import datetime as dt
+
+today = dt.date.today()
```

Comment:
Might want to use UTC date here. `dt.datetime.utcnow()` and then FMT it with `%Y-%m-%d`